04-onion-routing.md

```diff
@@ -724,6 +724,12 @@ General permanent failure of the processing node.
 
 The processing node has a required feature which was not in this onion.
 
+1. type: BADONION|PERM (`invalid_onion_payload`)
```

Comment:
I treat this as the default `BADONION` error, but maybe it would make more sense to assign it a specific bit (e.g. `BADONION|PERM|1` or `BADONION|PERM|7`).
Let me know what you think would be best.